### PR TITLE
Add CI on windows

### PR DESCRIPTION
**Pull request by dgageot**
Signed-off-by: David Gageot <david@gageot.net>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -29,3 +29,8 @@ jobs:
       go: "1.12.x"
       script:
         - make
+    - os: windows
+      go: "1.11.x"
```
Comment (nkubala) [inline .travis.yml]: does go 1.12 work on windows?

Comment (dgageot) [reply .travis.yml]: yes. I wanted to start small :-)

**Comment by dgageot** [comment]
@balopat @tejal29 @nkubala Do you think it's better to run the windows CI on the same service than osx and linux? Or it is better to use multiple services to lower the impact of an outage?

**Comment by nkubala** [comment]
+1, I think consolidating infrastructure > "redundancy" in case of an outage

**Pull request merged**